docker/build_support/.dockerignore

```diff
@@ -16,4 +16,5 @@ Dockerfile
 README.md
 LICENSE
 config.yml
-Makefile
\ No newline at end of file
+Makefile
+*.mk
```

Comment:
`.dockerignore` files simply remove files when COPIED into an image ot help keep them lean.  `.mk` files are only used in CI and development and can left from the resulting image.